docs/libs/ml/quickstart.md

```diff
@@ -24,4 +24,198 @@ under the License.
+functions for formatted data, such as the LibSVM format. For supervised learning problems it is
+common to use the `LabeledVector` class to represent the `(features, label)` examples. A `LabeledVector`
+object will have a FlinkML `Vector` member representing the features of the example and a `Double`
+member which represents the label, which could be the class in a classification problem, or the dependent
+variable for a regression problem.
+
+# TODO: Get dataset that has separate train and test sets
+As an example, we can use the Breast Cancer Wisconsin (Diagnostic) Data Set, which you can
+[download from the UCI ML repository](http://archive.ics.uci.edu/ml/machine-learning-databases/breast-cancer-wisconsin/breast-cancer-wisconsin.data).
+
+We can load the data as a `DataSet[String]` first:
+
+{% highlight scala %}
+
+val cancer = env.readCsvFile[(String, String, String, String, String, String, String, String, String, String, String)]("/path/to/breast-cancer-wisconsin.data")
+
+{% endhighlight %}
+
+The dataset has some missing values indicated by `?`. We can filter those rows out and
+then transform the data into a `DataSet[LabeledVector]`. This will allow us to use the
+dataset with the FlinkML classification algorithms.
+
+{% highlight scala %}
+
+val cancerLV = cancer
+  .map(_.productIterator.toList)
+  .filter(!_.contains("?"))
+  .map{list =>
+    val numList = list.map(_.asInstanceOf[String].toDouble)
+    LabeledVector(numList(11), DenseVector(numList.take(10).toArray))
+    }
+
+{% endhighlight %}
+
+We can then use this data to train a learner.
+
+A common format for ML datasets is the LibSVM format and a number of datasets using that format can be
+found [in the LibSVM datasets website](http://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/). FlinkML provides utilities for loading
+datasets using the LibSVM format through the `readLibSVM` function available through the MLUtils object.
+You can also save datasets in the LibSVM format using the `writeLibSVM` function.
+Let's import the Adult (a9a) dataset. You can download the 
+[training set here](http://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/binary/a8a)
+and the [test set here](http://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/binary/a8a.t).
+
+We can simply import the dataset then using:
+
+{% highlight scala %}
+
+val adultTrain = MLUtils.readLibSVM("path/to/a8a")
+val adultTest = MLUtils.readLibSVM("path/to/a8a.t")
+
+{% endhighlight %}
+
+This gives us a `DataSet[LabeledVector]` that we will use in the following section to create a classifier.
+
+Due to an error in the test dataset we have to adjust the test data using the following code, to 
+ensure that the dimensionality of all test examples is 123, as with the training set:
+
+{% highlight scala %}
+
+val adjustedTest = adultTest.map{lv =>
+      val vec = lv.vector.asBreeze
+      val padded = vec.padTo(123, 0.0).toDenseVector
```

Comment:
This will create a dense vector. Thus also `fvec` will be dense. Do we want that?